src/components/Dashboard.js

```diff
@@ -11,7 +13,7 @@ const Dashboard = (props) => {
     }, []);
     return (
     <ul>
-        {reviews.map((item) => (
+        {reviews.filter((item) => item.category === cat || !cat).map((item) => (
```

Comment:
we shouldn't be using a `.filter` here as the backend has endpoints to deal with this kind of request. We should be making another request our api here instead of requesting all the reviews constantly and filtering through them.

Reply:
I don't know what the endpoint would be, though?